packages/core/src/components/meter/meter.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.hasCenterTitle {
+    justify-content: center;
+  }
+
+  &.hasMarginTop {
+    margin-bottom: 0;
+    margin-top: var(--spacing-xsmall);
+  }
+
+  &.is-mobile {
+    font: var(--text-body-small);
+    font-weight: 300;
+    letter-spacing: var(--text-body-small-letter);
+  }
+
+  * {
```

Comment:
block: It is hazardous, so why do we need it?

Reply:
I added it to ensure that any component used in kids doesn't break the layout with padding, but I think it's better to leave it up to whoever uses it. 🤔

Reply:
Made: https://github.com/juntossomosmais/atomium/pull/695/commits/407764e7666b3a1d1244e0520940eaef4ad8b306